scanpipe/models.py

```diff
@@ -280,15 +280,15 @@ def get_root_content(directory):
         return [str(path.relative_to(directory)) for path in directory.glob("*")]
 
     @property
-    def input_root(self):
+    def input_root(self) -> list:
```

Comment:
Not sure we want to mix and match type annotations. We should probably remove this.